bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -504,6 +504,12 @@ public boolean isSecure() {
         };
     }
 
+    public static String buildStatsLoggerScopeName(BookieId addr) {
```

Comment:
The name of this method can be misleading. We already removed the bookie address from the metric names (instead it's added as a label). Sine this appears to be only by tests, we should just move the methods to tests folder.

Reply:
Addressed.